pkg/proxy/iptables/proxier_test.go

```diff
@@ -860,12 +860,12 @@ COMMIT
 -A KUBE-POSTROUTING -j MARK --xor-mark 0x4000
 -A KUBE-POSTROUTING -m comment --comment "kubernetes service traffic requiring SNAT" -j MASQUERADE
 -A KUBE-MARK-MASQ -j MARK --or-mark 0x4000
--A KUBE-SERVICES -m comment --comment "ns1/svc1:p80 cluster IP" -m tcp -p tcp -d 10.20.30.41/32 --dport 80 ! -s 10.0.0.0/24 -j KUBE-MARK-MASQ
+-A KUBE-SVC-XPGD46QRK7WJZT7O -m comment --comment "ns1/svc1:p80 cluster IP" -m tcp -p tcp -d 10.20.30.41/32 --dport 80 ! -s 10.0.0.0/24 -j KUBE-MARK-MASQ
```

Comment:
hm... in terms of making this easy-to-understand, it might be nice to output the lines in the opposite order, if that's not too hard. I suspect that would also make the logical flow in the generating code clearer.

Reply:
yes I understand the confusion, will change this since in any case, we sort them alphabetically when comparing.

Reply:
done.